docs/discover/document-explorer.asciidoc

```diff
@@ -1,10 +1,10 @@
 [[document-explorer]]
-== Document Explorer
+== Explore your documents
 
-beta::[]
 
-*Discover* has a *Document Explorer* with resizable columns, better data sorting and comparison,
-and a fullscreen view.
+*Discover* displays your your documents in table format, so you can
```

Comment:
Do you think that its worth to mention that the documents are paginated and users can configure the numbers of documents displayed on one page?